### ISSUE TITLE
Result is not sharp on lego. Any tip to improve quality?

### ISSUE THREAD

**Issue by pureexe** [ashawkey/torch-ngp#21]
I appreciate your great work! 

However, I train Lego scene with both [torch-ngp](https://github.com/ashawkey/torch-ngp) and [instant-ngp](https://github.com/NVlabs/instant-ngp) at 20000 steps (200 epoch). instant-ngp has a sharper result (green rectangle) while torch-ngp is missing some part of the object (red rectangle). Any tip to improve the quality? 

the training command is 
```
OMP_NUM_THREADS=8 CUDA_VISIBLE_DEVICES=0 python train_nerf.py data/nerf_synthetic/lego --workspace trial_nerf_lego_tcnn --fp16 --tcnn --cuda_ray --bound 1 --scale 0.8 --mode blender
```

![image](https://user-images.githubusercontent.com/1303847/155891248-c55d4b4a-324b-432d-b76d-5e6832e0c7f1.png)

Best regards

**Comment by ashawkey** [comment]
You are right, the rendering quality is not still reproduced as mentioned in https://github.com/ashawkey/torch-ngp/issues/17. Since there are lots of details in the original implementation, it may take some time for me to figure out the reason.
Currently, to improve quality (at the cost of speed), you can remove `--cuda_ray`, and enlarge `--num_steps` and `--upsample_steps` to use the pytorch ray marching.

**Comment by ashawkey** [comment]
The real reason is instant-ngp use all train/val/test set to train, which is set to default in GUI mode now. You can try it now!

**Comment by Dengzhi-USTC** [comment]
Hi, have you solve the problem about the ray marching?
